integration/fixtures/registries.conf

```diff
@@ -0,0 +1,26 @@
+[[registry]]
+url = "mirror.local"
+mirror = [
+    { url = "mirror-0.local" },
+    { url = "mirror-1.local" },
```

Comment:
All the `.local` host names could in principle exist. Maybe `*.invalid`, per RFC 2606?

Reply:
Alright, I changed them.